tests/python/test_mpm88.py

```diff
@@ -113,8 +113,11 @@ def test_mpm88():
     run_mpm88_test()
 
 
-# @pytest.mark.skipif(platform.system() == 'Windows',
-#                     reason='Stuck on Appveyor?')
+def _is_appveyor():
+    return int(os.getenv('IS_APPVEYOR', 0)) > 0
```

Comment:
```suggestion
    return bool(os.getenv('APPVEYOR'))
```
Please see https://www.appveyor.com/docs/environment-variables.